src/check_redis_cluster.py

```diff
@@ -32,24 +32,26 @@
 from argparse import ArgumentParser
 from sys import exit
 
+
 def main():
     """Main entrypoint for script"""
 
     args = get_parser().parse_args()
 
-    master_addr, master_state, cluster_state_master, failed_master = _get_cluster_status(
-        args.master_port, args.password)
-    slave_addr, slave_state, cluster_state_slave, failed_slave = _get_cluster_status(
-        args.slave_port, args.password)
+    master_addr, master_state, cluster_state_master, failed_master = \
+        _get_cluster_status(args.master_port, args.password)
```

Comment:
`\` is not a preferred way in PEP8 resp. braces is the preferred way for multilining. I am not completely sure if the below is the correct syntax, but try it with that:
```suggestion
    master_addr, master_state, cluster_state_master, failed_master = (
        _get_cluster_status(args.master_port, args.password)
    )
```